test/integration/integration_test.go

```diff
@@ -176,6 +176,11 @@ func FeatureContext(s *godog.Suite) {
 	s.Step(`^([^"]*) of command "minishift (.*)" (contains|does not contain) "(.*)"$`,
 		commandReturnContains)
 
+	// setting image caching operation
+	s.Step(`^image caching is (disabled|enabled)$`, minishift.setImageCaching)
```

Comment:
Here maybe the 2 line format we use in other definitions should be followed. (However in current 1 line format it is quite nicely readable. We might reconsider.)

Reply:
Don't have the strong opinion on it. My general rule is readability. 
With long lines, two lines is good but for smaller line one line is good.

Reply:
@budhrg True, lets keep it 👍